src/api-reference/reference.handlebars

```diff
@@ -148,7 +153,11 @@
                                                         <p>
                                                             <code>Authorization</code>
                                                             <a href="#" data-placement="top" data-toggle="tooltip" title="Required header"> <i class="fa fa-asterisk fa-danger"></i> </a>
+                                                        {{#if x-app-token}}
+                                                            <small>&bull; Equal to 'Bearer xxx', 'xxx' being the App authentication token</small>
```

Comment:
Add link to doc for getting an app token https://api.akeneo.com/apps/apps-getting-started.html